presto-hive/src/main/java/com/facebook/presto/hive/orc/DwrfPageSourceFactory.java

```diff
@@ -73,6 +75,10 @@ public DwrfPageSourceFactory(TypeManager typeManager, HdfsEnvironment hdfsEnviro
             return Optional.empty();
         }
 
+        if (fileSize == 0) {
+            throw new PrestoException(HIVE_BAD_DATA, "ORC file is empty: " + path);
```

Comment:
forgive my ignorance, should it be "DWRF file is empty"?

Reply:
That would be logical. However, we always use “ORC” in error messages, both because it’s easier, and that it’s generally referred to as “ORC” inside Facebook.

You now have me reconsidering if we should say “DWRF” here since empty files is actually a point of difference between the formats.

Reply:
so it was just my ignorance, thanks for explaining